Source/Scene/FeatureTable.js

```diff
@@ -312,4 +404,29 @@ FeatureTable.prototype.getPropertyTypedArray = function (propertyId) {
   return undefined;
 };
 
+/**
+ * Returns a typed array containing the property values for the property with the given semantic.
+ * <p>
+ * This only operates on the underlying {@link MetadataTable} (if present) as
+ * {@link JsonMetadataTable} and {@link BatchTableHierarchy} do not have
+ * semantics.
+ * </p>
+ *
+ * @param {String} semantic The case-sensitive semantic of the property.
+ * @returns {*} The typed array containing the property values or <code>undefined</code> if the property values are not stored in a typed array.
+ *
+ * @private
+ */
+FeatureTable.prototype.getPropertyTypedArrayBySemantic = function (semantic) {
```

Comment:
For the sake of naming consistency, should this be `getSemanticTypedArray`?

I see that we use `...BySemantic` quite a lot, but we deviate from that in `hasSemantic`.

Reply:
Resolving as we're back to the `xxxBySemantic` naming